lib/game.rb

```diff
@@ -217,12 +213,16 @@ def render_computer_output
 
   def render_player_output
     if false
-      puts "YOU SUNK A BOAT"
+      puts 'YOU SUNK A BOAT'
     elsif 1 == 2
-      puts "Your shot on #{computer_guess} was a hit."
+      puts "Your shot on #{@player_guess} was a hit."
     else
-      puts "Your shot on #{computer_guess} was a miss."
+      puts "Your shot on #{@player_guess} was a miss."
     end
   end
 
+  def output_shot_result
```

Comment:
this method solved the problem of the messages appearing too far apart on the screen